logging/include/RialtoLogging.h

```diff
@@ -39,6 +39,8 @@ extern "C"
         RIALTO_DEBUG_LEVEL_MILESTONE = (1u << 3),
         RIALTO_DEBUG_LEVEL_INFO = (1u << 4),
         RIALTO_DEBUG_LEVEL_DEBUG = (1u << 5),
+        RIALTO_DEBUG_LEVEL_EXTERNAL =
```

Comment:
Is there a reason why it needs to be last? If we want it in a fixed position why not add it add the start of the list?

Reply:
It's my suggestion only. It's last, because I'd like this level to be independent from RIALTO_DEBUG environment variable. Furthermore it's "unusual" level. I can remove comment in the code, if you don't like it.